scripts/gitlab/build_and_test.sh

```diff
@@ -151,4 +168,14 @@ then
     then
         echo "ERROR: failure(s) while running CTest" && exit 1
     fi
+
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    echo "~~~~~ CLEAN UP"
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    echo "~~~~~ Space used BEFORE clean:"
+    du -sh
+    make clean
+    echo "~~~~~ Space used AFTER clean:"
+    du -sh
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
```

Comment:
@kab163, I think this is not really useful in production, indeed. I understand the interest for debugging, but the information is less relevant in a daily use, and `du -sh` can be slow and introduce a delay in our pipelines.

Reply:
Agreed. Running du in a directory with a lot of files/subdirectories can take a long time. Perhaps, leave these lines in the script for debugging if needed, but commented out so they don't run otherwise.

Reply:
I would prefer if those were removed. The commit removing those could be reverted at will, and the syntax is not very complex anyway.

Reply:
Sounds good. Thanks for the feedback!